dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/ServletFilter.java

```diff
@@ -322,46 +320,88 @@ private String resolveHttpRoute(SRTServletRequest srtServletRequest, String cont
 					}
 				}
 				
+				/*
+				 * Explicitly deal with resources loaded by JSF / Jakarta Faces
+				 * 
+				 * URL for resources request will end with the file extension of the original requesting 
+				 * 
+				 * i.e., page.xhtml -> /jakarta.faces.resource/someFile.file.xhtml
+				 */
+				else if (servletPath.startsWith("/jakarta.faces.resource") || servletPath.startsWith("/javax.faces.resource")) {
+					String[] arr = servletPath.split("\\.");
+					String extension = arr[arr.length-1];
```

Comment:
Will the extensions have both upper case and lower case of the same thing?

Reply:
Upper case and lower case would be different servlet url mappings. `file.xhtml` != `file.XHTML`